aip/0123.md

```diff
@@ -73,6 +74,30 @@ message Topic {
 - Pattern variables (the segments within braces) **must** use `snake_case`, and
   **must not** use an `_id` suffix.
 
+#### Pattern uniqueness
+
+When multiple patterns are defined within a resource, these patterns **must**
+be mutually unique, where uniqueness is defined as being by-character identical
+once all resource ID path segments have been removed, leaving all `/`
+separators.
+
+Therefore the following two patterns **must not** be defined within the same
+resource:
+
+- `user/{user}`
+- `user/{user_part_1}~{user_part_2}`
+
+Because both patterns reduce to `user/` once the resource ID path segments have
+been removed.
+
+The following two patterns may be defined within the same resource:
```

Comment:
This still seems like **should not** to me, because AIP-122 frowns on this sort of thing. Rather than dwell on that, I propose striking from line 93 to the end of the section.

Reply:
done